fastlane/lib/fastlane/actions/slack.rb

```diff
@@ -155,7 +152,10 @@ def self.details
       # @!group Helper
       #####################################################
 
+      # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
class had already reached `Perceived Complexity` as my run of `bundle exec rubocop` as it was saying 19 out 18.